share/tools/web_config/sample_prompts/nim.fish

```diff
@@ -84,7 +84,6 @@ function fish_prompt
     end
 
     if test "$fish_key_bindings" = fish_vi_key_bindings
-        or test "$fish_key_bindings" = fish_hybrid_key_bindings
```

Comment:
If we did merge this, this line should stay, because people *will* keep using their old hybrid_key_bindings.